internal/api/fetch_files_timing.go

```diff
@@ -0,0 +1,84 @@
+package api
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"slices"
+)
+
+type fetchFilesTimingParams struct {
+	Paths []string `json:"paths"`
+}
+
+type fileTiming struct {
+	Path     string
+	Duration int
+}
+
+// FetchFilesTiming fetches the timing of the requested files from the server.
+// If the file is unknown to the server, it will have a duration of 0.
+// The files are sorted by duration in descending order.
+func (c Client) FetchFilesTiming(suiteSlug string, files []string) ([]fileTiming, error) {
+	url := fmt.Sprintf("%s/v2/analytics/organizations/%s/suites/%s/test_files", c.ServerBaseUrl, c.OrganizationSlug, suiteSlug)
+
+	requestBody, err := json.Marshal(fetchFilesTimingParams{
+		Paths: files,
+	})
+
+	if err != nil {
+		return nil, fmt.Errorf("converting params to JSON: %w", err)
+	}
+
+	req, err := http.NewRequest(http.MethodGet, url, bytes.NewBuffer(requestBody))
```

Comment:
This isn't going to work. Request body for GET will be ignored. The server currently expecting the files to be in query params, but it doesn't work either because the payload is too large to be a query params. We need to change the endpoint into a POST endpoint, or remove the files params altogether. @wooly